extra-misc/hollywood/autobuild/defines

```diff
@@ -0,0 +1,4 @@
+PKGNAME=hollywood
+PKGSEC=extra
+PKGDEP="apg bmon byobu cmatrix coreutils gawk htop man-db mlocate moreutils mplayer openssh speedometer tree util-linux"
```

Comment:
I doubt that all of these are actual dependencies, especially `byobu`, `mplayer`, ... I think these are supposed to serve as frontends? Please only include all those required for `hollywood` to run as dependencies, and all others should go to `PKGSUG` as optional dependencies.

Reply:
These packages are required to run as dependencies as the README shows.

Reply:
WIP

Reply:
I don't know where you got this from, nothing like that is mentioned in the README, unless you meant "sample Apps."